pom.xml

```diff
@@ -41,15 +43,17 @@
             </manifest>
           </archive>
         </configuration>
-        <version>3.2.2</version>
       </plugin>
       <plugin>
         <artifactId>maven-assembly-plugin</artifactId>
         <executions>
           <execution>
+            <!-- this is used for inheritance merges -->
+            <id>make-assembly</id>
             <goals>
-              <goal>attached</goal>
+              <goal>single</goal>
             </goals>
+            <!-- bind to the packaging phase -->
```

Comment:
Consistent with [the documentation](https://maven.apache.org/plugins/maven-assembly-plugin/usage.html). Needed to adapt to the newer version specified in the parent POM.